corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -745,6 +762,13 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             }
         },
 
+        updateSubmitButtonStatus: function (disabled) {
+            if (this.sidebarEnabled) {
+                sessionStorage.submitDisabled = disabled;
```

Comment:
Seeing this in sessionStorage makes me think it's meant to persist longer than the life of the view. Why not put it in a view-level variable?

Reply:
Re-reading the description, I now see there was a previous PR that I hadn't seen. Looking at that.

Reply:
Having read the other PR now, original question still stands.

Reply:
between SSCS search requests sometimes we want the var to be true and other times false so setting at the view level wouldn't make that possible

Reply:
I am not sure I follow why the view-level var would not work. Are we destroying the view and rebuilding it?

Reply:
@MartinRiese yes that's right it's rebuilt each time (feel free to reference these slides for SSCS: https://docs.google.com/presentation/d/14acWNxMEGgT83k0rtorypaNlivXlxOcP4IqNyyru3pw/edit#slide=id.g1e71313ea23_0_0)